packages/core/src/utils/__tests__/shortenedUrl.test.ts

```diff
@@ -0,0 +1,114 @@
+import type { Response } from 'node-fetch'
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Headers } from 'node-fetch'
```

Comment:
If this doesn't work we should add it to the dev dependencies

```suggestion
import type { Headers } from 'node-fetch'
```